functions-and-scope.js

```diff
@@ -41,6 +65,23 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 6.642857142857143
+// STAPPENPLAN //
+// Hoe wordt een gemiddelde berekend? // AANTAL WAARDES OPTELLEN -/- EN DELEN DOOR het AANTAL
+// Wat moet ik verzamelen uit de array van cijfers om uiteindelijk een gemiddelde te kunnen berekenen? // WAARDES
+// Hoe zorgt ik ervoor dat ik alle waardes uit de array kan langslopen, ook als de array wel 100 entries zou bevatten? // FOR LOOP
+
+console.log("------------Opdracht 2------------------");
+
+let sum = 0;
+let counter = 0;
+
+for (let i = 0; i < grades.length; i++) {
+    counter = counter + 1;
+    sum += grades[i];
+}
+const averageGrades = sum / counter;
```

Comment:
Je hebt de hele `counter` variabele niet nodig, omdat die altijd even groot is als `grades.length`.
```suggestion
const averageGrades = sum / grades.length;
```